addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -2254,20 +2254,7 @@ public Action:OnRoundStart(Handle:event, const String:name[], bool:dontBroadcast
 		}
 	}
 
-	if(blueBoss)
-	{
-		SetTeamScore(_:TFTeam_Red, GetTeamScore(OtherTeam));
-		SetTeamScore(_:TFTeam_Blue, GetTeamScore(BossTeam));
-		OtherTeam=_:TFTeam_Red;
-		BossTeam=_:TFTeam_Blue;
-	}
-	else
-	{
-		SetTeamScore(_:TFTeam_Red, GetTeamScore(BossTeam));
-		SetTeamScore(_:TFTeam_Blue, GetTeamScore(OtherTeam));
-		OtherTeam=_:TFTeam_Blue;
-		BossTeam=_:TFTeam_Red;
-	}
+	SetBossTeam(blueBoss ? TFTeam_Blue : TFTeam_Red);
```

Comment:
Why not just pass `blueBoss` here?

`SetBossTeam` is also misleading since it does more than that.